libyul/optimiser/UnusedFunctionsCommon.h

```diff
@@ -59,45 +59,6 @@ FunctionDefinition createLinkingFunction(
 	YulString const& _originalFunctionName,
```

Comment:
While we're at changing this, can you document what these helpers are doing in this header?

Reply:
Fixed